app/components/home/contents/HomeContent.tsx

```diff
@@ -383,7 +383,10 @@ export default function HomeContent() {
 
         try {
           // If direct playback fails, try converting raw PCM to WAV
-          const wavBuffer = createWavFile(pcmData)
+          const wavBuffer = createWavFile(
+            pcmData,
+            interaction.sample_rate || 16000,
+          )
```

Comment:
Does raw PCM playback usually fail?

Reply:
I think it's fine for now. It's more or less a catch to see if we can play it as is first before attempting to create the wav.